Introduce a dummy version GLIBC_2.28, so that the compat symbol checks
give the right result for the glibc 2.27 backport (i.e., the symbols are
still built into 2.27 libraries).

This is not needed upstream because of the 2.28 version number there.

diff --git a/crypt/Versions b/crypt/Versions
index 389e7d544aff1e2e..59259c6a397d915e 100644
--- a/crypt/Versions
+++ b/crypt/Versions
@@ -2,4 +2,7 @@ libcrypt {
   GLIBC_2.0 {
     crypt; crypt_r; encrypt; encrypt_r; fcrypt; setkey; setkey_r;
   }
+  GLIBC_2.28 {
+    # Dummy entry to make conditionals for compat symbols work.
+  }
 }
